### PR TITLE
[CFP-297] Add Google Tag Manager

### PR DESCRIPTION
**Pull request by naseberry**
#### What

Install Google Tag Manager

#### Ticket

[Install Google Tag Manager](https://dsdmoj.atlassian.net/browse/CFP-297)

#### Why

-

### CODE REVIEW

**Review by imtiazAhmd — APPROVED** [review]

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -35,9 +35,10 @@
     %a.govuk-skip-link{ href: '#main-content' }
       = t('layouts.skip_content')
 
-    - if GoogleAnalytics::DataTracking.tag_manager? && !@disable_analytics
-      %noscript
-        %iframe{ src: "https://www.googletagmanager.com/ns.html?id=#{ENV['GTM_TRACKED_ID']}", style: 'display: none; visibility:hidden', height: '0', width: '0' }
+    - if GoogleAnalytics::DataTracking.enabled? && !@disable_analytics
+      - if GoogleAnalytics::DataTracking.tag_manager?
```
Comment (jrmhaig) [inline app/views/layouts/application.html.haml:39]: `tag_manager?` already includes `enabled?`. Is there a reason why this needs to be broken into two lines?

Comment (jrmhaig) [inline app/views/layouts/application.html.haml]: `tag_manager?` already includes `enabled?`. Is there a reason why this needs to be broken into two lines?

**Review by jsugarman — APPROVED** [review]
looks fine to me, although as joe mentions there may be a way to combine the conditionals on the application layout

**Review by jessicabrentnall — APPROVED** [review]

File: app/views/layouts/application.html.haml (hunk 1 above)
Comment (naseberry) [reply app/views/layouts/application.html.haml]: You're correct, this has been corrected

**Review by jrmhaig — APPROVED** [review]

**Pull request merged**